### ISSUE TITLE
Narrator announces Invalid word for the empty edit box "Forecast Configuration".

### ISSUE THREAD

**Issue by fsteffi** [OfficeDev/office-ui-fabric-react#11731]
Environment Details:​​
Browser Versions: Microsoft Edge 44.18362.387.0,
Chrome Version 77.0.3865.120 (Official Build) (64-bit)
Windows Version: Windows10​​​
NVDA 2019.2.1

Repro Steps:​​
1. Launch the application URL 
2. Select application as "Sales hub" from the list of application.
3. From the change area which is present in the bottom select "App Settings"
4. Navigate to "Site Map" and then Navigate to "Goal Management" and select "Forecast Configuration"
5. Fast Track Your Forecast Screen will appears.
6. Click on "Org chart forecast"
7. Forecast Configuration screen will appears
8. Start narrator and navigate to "Forecast configuration edit box".

Actual:​
Narrator announces wrong information when we are moving to the "Forecast configuration edit box", It announces that invalid word when the focus on edit box.

Same issue is observed with NVDA.

Expected:​​
Narrator should announce invalid word it should give information like empty/Blank edit box.

User Impact:​
Screen reader user will think that we enters something which is not valid and he/she will get confused..

![NewIssue](https://user-images.githubusercontent.com/24580570/72523032-f70ec400-3884-11ea-959b-899c249a421e.png)

**Comment by fsteffi** [comment]
Same issue observed here https://fabricweb.z5.web.core.windows.net/oufr/7.22.1/#/examples/textfield

**Comment by xugao** [comment]
@fsteffi - for future reference, pls no need to include application specific repro steps in the issue. here is a good example for similar bug report: https://github.com/OfficeDev/office-ui-fabric-react/issues/11425

**Comment by fsteffi** [comment]
@xugao You can hit the below url https://fabricweb.z5.web.core.windows.net/oufr/7.22.1/#/examples/textfield
and please try to navigate to "With error message text field" Start narrator and navigate to "With error message text field"

Actual:​
Narrator announces wrong information when we are moving to the text field It announces that invalid word when the focus on edit box.

Same issue is observed with NVDA.
Expected:​​
Narrator should announce invalid word it should give information like empty/Blank edit box.

**Comment by xugao** [comment]
@fsteffi ,

Here is the actual behavior I see, which seems to be correct:
Narrator +  Edge: With error message, edit, Invalid, end of line
NVDA + Chrome: With error message, invalid entry, blank

Could you clarify what exactly you would expect here?

**Comment by fsteffi** [comment]
@xugao when there is no word (in the edit box)present then also narrator and nvda announce invalid word which is not correct.

**Comment by xugao** [comment]
@fsteffi - In the "With error message" example, we are trying to show how to use `errorMessage`. Therefore the text field is invalid regardless what input is currently inside (empty or non-empty). it's up to the user to determine if the field is invalid/should show error message or not in real use cases

**Comment by fsteffi** [comment]
@xugao Can you please mark this as by design??
